### PR TITLE
[Bulk] Validation Fixes

### PR DESCRIPTION
**Pull request by AqMoh**
This PR fixes 46 issues.

### CODE REVIEW

File: concepts/changelog-archive.md [hunk 1]
```diff
@@ -915,76 +915,76 @@ Added new APIs and resources for subscribing to and retrieving call records from
...
 ### Identity and access
 
 | **Change type** | **Version**   | **Description** |
 |:---|:---|:---|
 | Addition | beta and v1.0 | Added the **authenticationRequirement** property and the **riskEventTypes_v2** property to the [signin](/graph/api/resources/signin?view=graph-rest-1.0&preserve-view=true) entity. |
-| Addition | beta and v1.0 | Added support for [delta queries for orgContact](/graph/api/orgcontact-delta?view=graph-rest-1.0).|
+| Addition | beta and v1.0 | Added support for [delta queries for orgContact](/graph/api/orgcontact-delta?view=graph-rest-1.0&preserve-view=true).|
 | Addition | v1.0 | Added new user permission [User.ManageIdentities.All](./permissions-reference.md#user-permissions).|
-| Addition | beta | Added the PrivilegedAccess.Read.AzureResources application permission for [Privileged Identity Management - Azure resources](/graph/api/resources/privilegedidentitymanagement-resources?view=graph-rest-beta). |
+| Addition | beta | Added the PrivilegedAccess.Read.AzureResources application permission for [Privileged Identity Management - Azure resources](/graph/api/resources/privilegedidentitymanagement-resources?view=graph-rest-&preserve-view=true). |
```
Comment (jmart1428) [inline concepts/changelog-archive.md:946]: Looks like "beta" got cut off

File: concepts/changelog-archive.md [hunk 2]
```diff
@@ -915,76 +915,76 @@ Added new APIs and resources for subscribing to and retrieving call records from
...
+| Addition        | v1.0       | Added the **subscriptions** navigation property to the [list](/graph/api/resources/list?view=graph-rest-1.0&preserve-view=true) entity. |
 
 ### Teamwork
 
 | **Change type** | **Version** | **Description**                          |
 | :-------------- | :---------- | :--------------------------------------- |
-| Addition | Beta |Added new property **supportedEntities** to [workforceintegration](/graph/api/resources/workforceintegration?view=graph-rest-beta) entity.|
-| Addition | v1.0 | Added new entity [teamwork](/graph/api/resources/teamwork?view=graph-rest-1.0).
-| Addition | Beta | Added support for ChannelMessage.Read.All to APIs for reading [chatMessages](/graph/api/resources/chatmessage?view=graph-rest-beta) in channels. |
+| Addition | Beta |Added new property **supportedEntities** to [workforceintegration](/graph/api/resources/workforceintegration?view=graph-rest-&preserve-view=true) entity.|
```
Comment (jmart1428) [inline concepts/changelog-archive.md:985]: Looks like "beta" got cut off.

**Comment by AqMoh** [comment]
@Lauragra @Danielabom 
The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you'd like to see. Thank you!

**Comment by v-emilypr** [comment]
#please-close

**Pull request closed without merging**